content/en/core/releases/4.12.0.md

```diff
@@ -0,0 +1,131 @@
+
+---
+title: "4.12.0 release notes"
+linkTitle: "4.12.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.12.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+This release includes changes to the user interface which will improve the learnability of the CHT by aligning with the look and feel of other android apps. Below is a summary of the changes, all of which were covered in the [July Round-up Call](https://youtube.com/watch?v=NSFRQt378Ls&t=460s):
+- [Update to the overall CHT app Navigation experience](https://forum.communityhealthtoolkit.org/t/the-ui-evolution-of-the-cht/3515#navigation-6)
+- Added the ability to temporarily [opt out](https://docs.communityhealthtoolkit.org/building/guides/updates/feature-flags/) of the new Navigation experience 
```

Comment:
```suggestion
- Added the ability to temporarily [opt out]({{< relref "building/guides/updates/feature-flags" >}}) of the new Navigation experience 
```

Minor, but I think we should use the short-code versions of these links to other pages on the docs site. The main benefit, as I understand it, is that at build time Hugo will detect if the shortcode link is broken (e.g. if the page was moved or removed).